dashboards/google-backupdr/backupdr-monitoring.json

```diff
@@ -0,0 +1,162 @@
+{
+  "displayName": "Backup and DR Monitoring",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "Backup and DR - Storage Pool Utilization",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "filter": "metric.type=\"staging-backupdr.sandbox.googleapis.com/backup_recovery_appliance/storage_pool/utilization\" resource.type=\"staging-backupdr.sandbox.googleapis.com/ManagementConsole\"",
```

Comment:
Let's also use the production metric types and resource type, as this dashboard won't be usable externally without that.

Reply:
As discussed, going to keep it to point to our sandbox env, Will update later pointing to prod end point once our services published there